recipes/langgraph-checkpoint-aws/meta.yaml

```diff
@@ -17,11 +16,11 @@ build:
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python >=3.9
```

Comment:
Please keep these. My change suggestion was just to remove the pre-defined variable, not to hard-code it.